flamingo/plugins/feeds.py

```diff
@@ -34,7 +34,9 @@ def pre_build(self, context):
                 if feed_config['type'] in ['rss', 'podcast']:
                     fg.description(feed_config['description'])
                     fg.link(href=feed_config['link'], rel='self')
-                    fg.link(href=feed_config['link_alternate'], rel='alternate')
+                    fg.link(
+                        href=feed_config['link_alternate'], rel='alternate'
+                    )
```

Comment:
This wrap is based on 80 chars? For most other projects we have been moving to 119/120 chars recently :-) We could consider to do that here, too.

Reply:
Yes. I don't insist on 80 chars per line. That's just what flake8 currently checks for and I wanted to make the tests pass.

Reply:
I don't subject to this change. But maybe we could increase the limit later on. Thx :-)